openhcl/virt_underhill/src/processor/tdx/mod.rs

```diff
@@ -873,15 +886,19 @@ impl UhProcessor<'_, TdxBacked> {
                     VmcsField::VMX_VMCS_EOI_EXIT_2,
                     VmcsField::VMX_VMCS_EOI_EXIT_3,
                 ];
-                for ((&field, eoi_exit), tmr) in fields
+                for ((&field, eoi_exit), (i, tmr)) in fields
                     .iter()
                     .zip(&mut self.backing.eoi_exit_bitmap)
-                    .zip(tmr.chunks_exact(2))
+                    .zip(tmr.chunks_exact(2).enumerate())
                 {
                     let tmr = tmr[0] as u64 | ((tmr[1] as u64) << 32);
                     if *eoi_exit != tmr {
                         self.runner.write_vmcs64(Vtl::Vtl0, field, !0, tmr);
                         *eoi_exit = tmr;
+                        // ipi_offload_tmr tracks the eoi_exit bitmap
```

Comment:
this comment makes no sense to me - this is because the kernel needs to know this info right? could we clarify that?

Reply:
Clarified some in the new version, let me know if it is still unclear